data_prep/roberta_graph_dataset.py

```diff
@@ -1,24 +1,61 @@
+import nltk
+nltk.download('punkt')
+from nltk import word_tokenize
```

Comment:
Why did you choose this tokenizer? I guess we could just use any tokenizer but I was wondering if this is the same tokenizer that is used in TfidfVectorizer (https://github.com/fvlieshout/ATCS_group1/blob/01dac384363bb858e8f69343bb7df45773f36079/data_prep/graph_utils.py#L113). It should right? Otherwise the tf-idf score is computed over a different vocabulary then the actual word nodes

Reply:
The tf.idf tokenizer is set to be `lambda x: x` which is the identity map (maps `x` to `x`). So tf.idf does not use any tokenizer since it already gets the text tokenized

Reply:
Oh and I just chose it because we used it in practical one and it's better than just using `split()`